### ISSUE TITLE
MongoDB - Service 'db' is undefined.

### ISSUE THREAD

**Issue by persello** [vapor/template#48]
Hello, when I create a new project with a mongoDB database, running `docker-compose build` gives me the following error:

```
ERROR: Service 'app' depends on service 'db' which is undefined.
``` 

### To Reproduce

Here is the sequence of commands I used.

```
vapor new project-name
y (Fluent: yes)
4 (db: Mongo)
n (Leaf: no)
cd project-name/
docker-compose build
```

### Expected behavior

I don't know much about Docker, but I think the project should build "out of the box".
Maybe related to issue #31.

### Environment

* Vapor Framework version: 4.45.5
* Vapor Toolbox version: 18.3.3
* OS version: macOS 11.3.1 (arm64)

**Comment by 0xTim** [comment]
Yeah we don't have an option for MongoDB in the compose file at the moment. @Joannis what are good options/settings for a default `mongodb` container?

**Comment by Joannis** [comment]
I've got a docker-compose setup that I commonly use, it's also used in the RW tutorials. @0xTim can I just paste it here?

**Comment by 0xTim** [comment]
@Joannis sure

**Comment by Joannis** [comment]
[mongo-setup.zip](https://github.com/vapor/template/files/6539538/mongo-setup.zip)

**Comment by dotmain** [comment]
This gist should resolve the matter with mongo.

https://gist.github.com/dotmain/514a71e782417fe09b0cfaa112d3c214

**Comment by 0xTim** [comment]
Resolved by #52
